sdk/eventgrid/azure-eventgrid/tests/perfstress_tests/send.py

```diff
@@ -0,0 +1,69 @@
+import random
+import asyncio
+from azure_devtools.perfstress_tests import PerfStressTest
+
+from azure.eventgrid import EventGridPublisherClient as SyncPublisherClient, EventGridEvent
+from azure.eventgrid.aio import EventGridPublisherClient as AsyncPublisherClient
+
+from azure.core.credentials import AzureKeyCredential
+
+class EventGridPerfTest(PerfStressTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+
+        # auth configuration
+        topic_key = self.get_from_env("EG_ACCESS_KEY")
+        endpoint = self.get_from_env("EG_TOPIC_HOSTNAME")
+
+        # Create clients
+        self.publisher_client = SyncPublisherClient(
+            endpoint=endpoint,
+            credential=AzureKeyCredential(topic_key)
+        )
+        self.async_publisher_client = AsyncPublisherClient(
+            endpoint=endpoint,
+            credential=AzureKeyCredential(topic_key)
+        )
+
+        services = ["EventGrid", "ServiceBus", "EventHubs", "Storage"]
+        self.event_list = []
+        for _ in range(self.args.num_events):
+            self.event_list.append(EventGridEvent(
+                event_type="Contoso.Items.ItemReceived",
+                data={
+                    "services": random.sample(services, k=random.randint(1, 4))
```

Comment:
This will test a different data set every time - we should probably keep each test consistent with throughput.
If you expect there to be variation in consuming each of these values that you want to compare - you could make it a cmd flag:
```python
parser.add_argument('-e', '--event-service', nargs='?', type=str, help='The event service type. Default is "EventGrib"', default='EventGrid')
```
But otherwise I think we can just stick to a single value.

Reply:
sticking to single value should be good